src/components/Main/Results/DeterministicLinePlot.tsx

```diff
@@ -348,7 +348,7 @@ export function DeterministicLinePlot({
                   <Scatter key={d.key} dataKey={d.key} fill={d.color} name={d.name} isAnimationActive={false} />
                 ))}
 
-                {translatePlots(t, linesToPlot).map((d) => (
```

Comment:
Translation has already been performed, so this function is redundant and can be removed.